source/Camera/Basic/C++Halcon/CmakeLists.txt

```diff
@@ -0,0 +1,84 @@
+cmake_minimum_required(VERSION 3.3.2 FATAL_ERROR)
+
+# Set project name and language
+project(
+    C++Halcon
+    CXX
+)
+
+SET(CMAKE_BUILD_TYPE Release)
+
+# Set C++ standard
+set(
+    CMAKE_CXX_STANDARD 
+    17
+)
+
+set(SAMPLES
+	HalconConnectUsingGenICam/HalconConnectUsingGenICam
+	HalconConnectUsingZivid/HalconConnectUsingZivid)
+
+# Look for Find*.cmake files in root directory
+set(
+    CMAKE_MODULE_PATH 
+    ${CMAKE_MODULE_PATH} 
+    ${CMAKE_CURRENT_SOURCE_DIR}
+)
+
+# ==================================================
+#  Zivid
+# ==================================================
+find_package(
+    Zivid
+    1.8.1
+    REQUIRED
+    COMPONENTS Core
+)
+
+# ==================================================
+#  OpenCV
```

Comment:
Is OpenCV needed for these samples? What for?
If it is, you need to explain this dependency in the main Readme.

Reply:
No, i will remove this

Reply:
done